mysensors/mysensors.py

```diff
@@ -62,6 +62,12 @@ def _handle_set(self, msg):
                 msg.child_id, msg.sub_type, msg.payload)
             self.alert(msg.node_id)
 
+    def _handle_req(self, msg):
+        if self.is_sensor(msg.node_id, msg.child_id):
+            value = self.sensors[msg.node_id].get_child_value(msg.child_id, msg.sub_type)
+            if value:
+                return msg.copy(type=MessageType.set, payload=str(value))
```

Comment:
The value should already be stored as a string, so you don't need to convert it. I've removed the global MessageType on the dev branch. There you can use self.const.MessageType.set, in the Gateway class.